### DISCUSSION TITLE
[ideas] Add command to set environment in Administration mode

### DISCUSSION THREAD

**Discussion by sanjeev40084** [microsoft/powerplatform-vscode#617] [Ideas]
Please update 'pac org' command to allow user to put environment in Administration mode. Since now we have list-settings attribute, may be it could be added there, something like this:

`pac org update-settings -n isadministrationmode -v true`

**Comment by sanjeev40084** [comment]
@petrochuk @snizar007 i believe in last office hour, someone said this option will be available in next release. since there is new release available this morning, do you know if this made the cut?

**Reply by petrochuk** [reply]
We tried implementing it but run into some issues with back-end API and had to remove it from the release.

**Reply by sanjeev40084** [reply]
@petrochuk thanks for the update.

**Comment by sanjeev40084** [comment]
this is available now by running 'set-runtime-state' command. (available from [pac cli 1.30.6](https://www.nuget.org/packages/Microsoft.PowerApps.CLI/1.30.6#releasenotes-body-tab))

`pac admin set-runtime-state -env 'env-id' -rs 'AdminMode'`
